enarx-keep-sgx-shim/src/event.rs

```diff
@@ -16,29 +17,38 @@ pub extern "C" fn event(
     aex: &mut StateSaveArea,
     ctx: &Context,
 ) {
-    match unsafe { core::slice::from_raw_parts(aex.gpr.rip as *const u8, 2) } {
-        // syscall
-        [0x0f, 0x05] => {
-            let syscall = aex.gpr.rax.into();
-            let mut h = Handler::new(tcs, aex, ctx);
+    // Exception Vector Table
+    match aex.gpr.exitinfo.exception().unwrap() {
```

Comment:
We can't use `unwrap()` in the shim. It will cause an infinite loop if it doesn't succeed.

Also, please note that our `exception()` function before was theoretical. We aren't actually sure if it is correct. So we definitely need to test it.